docs/source/markdown/podman-image-trust.1.md

```diff
@@ -30,8 +30,8 @@ If no configuration is found for any of these scopes, the default value (specifi
 
 Trust **type** provides a way to:
 
-Whitelist ("accept") or
-Blacklist ("reject") registries or
+Allowlist ("accept") or
+Denylist ("reject") registries or
```

Comment:
are there code changes for this, or did they happen in a separate PR?

Reply:
No code changes